packages/cozy-clisk/src/launcher/saveFiles.js

```diff
@@ -504,6 +464,15 @@ function attachFileToEntry(entry, fileDocument) {
   return entry
 }
 
+/**
+ * Get the full path of a given file, from it's folder path and file or entry
+ *
+ * @param {Object} arg - arg option object
+ * @param {import('cozy-client/types/types').FileDocument} [arg.file] - io.cozy.files object
+ * @param {saveFilesEntry} [arg.entry] - saveFiles entry
+ * @param {saveFileOptions} arg.options - io.cozy.files object
+ * @returns {string | undefined} - file full path
```

Comment:
this is not part of this PR, but its weird to return undefined from this kind of method.